packages/babel-plugin-proposal-object-rest-spread/test/fixtures/object-rest/object-ref-computed/input.js

```diff
@@ -0,0 +1,20 @@
+var key, x, y, z;
```

Comment:
This test case is revised from https://github.com/babel/babel/blob/0b3f883ed19b4958449f3dbf0f6104b4eeda570b/packages/babel-plugin-proposal-object-rest-spread/test/fixtures/object-rest/impure-computed/input.js#L1